src/css/styles.css

```diff
@@ -56,8 +56,7 @@
   border-radius: 0;
 }
 
-.villain--static .villain-canvas,
-.villain--static .villain-canvas--overlay {
```

Comment:
Are you sure this is not used ?

Reply:
`.villain-canvas--overlay` is not used anywhere in code. I checked this four times :)

Reply:
Lol, sorry no idea why I left that then :stuck_out_tongue: